client/my-sites/domains/domain-management/list/index.jsx

```diff
@@ -437,23 +355,25 @@ export class List extends React.Component {
 				this.setState( {
 					settingPrimaryDomain: false,
 					changePrimaryDomainModeEnabled: false,
-					notice: {
-						type: PRIMARY_DOMAIN_CHANGE_SUCCESS,
-						domainName: domain.name,
-						previousDomainName: currentPrimaryName,
-					},
 				} );
+
+				this.props.successNotice(
+					translate(
+						'Primary Domain changed: All domains will redirect to {{em}}%(domainName)s{{/em}}.',
+						{ args: { domainName: domain.name }, components: { em: <em /> } }
+					),
+					{ duration: 10000, isPersistent: true }
+				);
 			},
-			error => {
+			() => {
 				this.setState( {
 					settingPrimaryDomain: false,
 					primaryDomainIndex: currentPrimaryIndex,
-					notice: {
-						type: PRIMARY_DOMAIN_CHANGE_FAIL,
-						domainName: domain.name,
-						error,
-					},
 				} );
+				this.props.errorNotice(
+					translate( "Something went wrong and we couldn't change your Primary Domain." ),
```

Comment:
Like above, I'd just keep `primary domain` lowercase - it's not a name of a special product, just a domain that is primary.

Reply:
This one too. I prefer it lowercase too.